tools/Google.Cloud.Docs.Snippets/UpdateMaskSnippets.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright 2025 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.SecretManager.V1;
+using Google.Protobuf.WellKnownTypes;
+using Grpc.Core;
+
+namespace Google.Cloud.Tools.Snippets
+{
+    public class UpdateMaskSnippets
+    {
+        public async Task UpdateMasks()
+        {
+            string projectId = "your-project-id";
+            string secretId = "test-secret";
+
+            // Sample: UpdateMasks
+            // Required using directives:
+            // using Google.Cloud.SecretManager.V1;
+            // using Google.Protobuf.WellKnownTypes;
+            // using Grpc.Core;
+
+            // Setup Client
+            SecretManagerServiceClient client = await SecretManagerServiceClient.CreateAsync();
+
+            // Prepare the resource with NEW values
+            Secret secret = new Secret
+            {
+                // Set the full resource name
+                SecretName = SecretName.FromProjectSecret(projectId, secretId),
+
+                // Populate ONLY the fields we intend to change
```

Comment:
Is this true? I thought that the best thing about field masks is that even if you have several values populated, only the ones in the field mask are updated.